prow/cmd/deck/main.go

```diff
@@ -684,6 +684,12 @@ func renderSpyglass(sg *spyglass.Spyglass, cfg config.Getter, src string, o opti
 		tgLink = ""
 	}
 
+	extraLinks, err := sg.ExtraLinks(src)
+	if err != nil {
+		logrus.WithError(err).WithField("page", src).Warn("Unexpectedly failed to fetch extra links")
```

Comment:
I think unexpectedly is unnecessary here. We add this in unit tests primarily to distinguish between "this scenario should produce and error, but didn't" and "this scenario should have passed but failed"

Reply:
I included "unexpected" because there are a lot of ways for it to fail that are expected and we don't care about. I remove it though.